src/styles/blocks/calendar.scss

```
@@ -0,0 +1,58 @@
+.calendar {
+  padding: $padding;
+  display: flex;
+  justify-content: center;
+
+  @each $name, $num in $days-in-week {
+    &--start-day--#{$name} > {
+      .calendar__sheet > :first-child {
+        margin-left: $cell-size * ($num - 1) + $gap-size * ($num - 1);
+      }
+    }
+  }
+
+  @for $n from 28 to 32 {
```

```suggestion
  @for $n from 28 through 31 {
```
It will work the same. but looks better and easier to undestand